### PR TITLE
Address comments for the KEP review

### PR DESCRIPTION
**Pull request by Dyanngg**
- Changed a couple of API types based on discussion
- Replaced `Authorize` action with `Empower` based on KEP comments

### CODE REVIEW

File: keps/sig-network/2091-cluster-network-policy/README.md [hunk 1]
```diff
@@ -434,13 +435,13 @@ type ClusterNetworkPolicyPeer struct {
 }
 
 const (
-	// RuleActionAuthorize is the highest priority allow rules which enables admins to provide exceptions to deny rules.
-	RuleActionAuthorize RuleAction = "Authorize"
+	// RuleActionEmpower is the highest priority allow rules which enables admins to provide exceptions to deny rules.
```
Comment (abhiraut) [inline keps/sig-network/2091-cluster-network-policy/README.md:438]: maybe reword this to remove "allow" since its not a allow but more of a bypass now..

File: keps/sig-network/2091-cluster-network-policy/README.md [hunk 2]
```diff
@@ -485,10 +486,11 @@ type ClusterDefaultNetworkPolicyIngress/EgressRule struct {
...
 
 type ClusterDefaultNetworkPolicyPeer struct {
-	PodSelector  *metav1.LabelSelector
-	// required if a PodSelector is specified
-	Namespaces   *Namespaces
-	IPBlock      *networkingv1.IPBlock
+	PodSelector       *metav1.LabelSelector
+	// one of NamespaceSelector or Namespaces is required, if a PodSelector is specified
+	NamespaceSelector *metav1.LabelSelector
+	Namespaces        *Namespaces
```
Comment (abhiraut) [inline keps/sig-network/2091-cluster-network-policy/README.md:492]: also need to mention that `NamepsaceSelector` and `Namespaces` are mutually exclusive

File: keps/sig-network/2091-cluster-network-policy/README.md (hunk 1 above)
Comment (Dyanngg) [reply keps/sig-network/2091-cluster-network-policy/README.md]: Done

File: keps/sig-network/2091-cluster-network-policy/README.md (hunk 2 above)
Comment (Dyanngg) [reply keps/sig-network/2091-cluster-network-policy/README.md:494]: Done in the latest commit

File: keps/sig-network/2091-cluster-network-policy/README.md [hunk 3]
```diff
@@ -510,6 +514,7 @@ type Namespaces struct {
 	// Labels are set only when scope is "SameLabels".
 	Labels      []string
 	// Selector is only set when scope is "Selector".
+	// Namespaces.Selector has the same effect as NamespaceSelector.
```
Comment (abhiraut) [inline keps/sig-network/2091-cluster-network-policy/README.md:517]: we can keep it for now; but i wonder if we need this strategy now or let users use NamespaceSelector
